sonar-project.properties

```
@@ -0,0 +1,7 @@
+sonar.projectName=digital-marketplace
```

```suggestion
sonar.projectName=digital_marketplace
```
I'd lean to keeping the `projectName` the same as the repo name for clarity.